README.md

```diff
@@ -25,7 +25,7 @@
 # 📖 Blog App <a name="about-project"></a>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/215249923-1cbd18df-fedb-4072-bdf0-55ae1c541e09.png)
**[Optional]**
- [ ] Your MIT link is not working :fearful: Kindly fix this link to make your readme file descriptive and professional :+1: